.github/workflows/socrata_upload.yaml

```diff
@@ -32,27 +32,31 @@ on:
       by_township:
         type: boolean
         description: Chunk Socrata upload by township
-        default: false
         required: false
 
 jobs:
   upload-open-data-assets:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
**[Question, non-blocking]** What's the story here?

Reply:
I was struggling with the github action telling me it couldn't find most versions of python. I came across some advice suggesting that the version of ubuntu was the culprit and it worked. I'll try again with latest and see if this solution is still needed.

Reply:
Okay, seems to no longer be an issue.